packages/audio/fluidsynth/package.mk

```diff
@@ -3,11 +3,11 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="fluidsynth"
-PKG_VERSION="1.1.6"
-PKG_SHA256="d28b47dfbf7f8e426902ae7fa2981d821fbf84f41da9e1b85be933d2d748f601"
+PKG_VERSION="2.1.5"
```

Comment:
this package is only used for `audiodecoder.fluidsynth` it's likely that bumping this may break that. Please test.

Reply:
Will raise in a seperate PR